src/highlighters.cc

```diff
@@ -519,16 +520,26 @@ UniquePtr<Highlighter> create_line_highlighter(HighlighterParameters params, Hig
 }
 
 const HighlighterDesc column_desc = {
-    "Parameters: <value string> <face>\n"
-    "Highlight the column given by evaluating <value string> with <face>",
-    {}
+    "Parameters: [-ruler <character>] <column> <face>\n"
+    "Highlight the column <column> with <face>",
+    {
+        { { "ruler", { ArgCompleter{}, "replace empty or whitespace cells with the given character. When provided, <face> is not applied to non-empty cells" } } },
+        ParameterDesc::Flags::SwitchesOnlyAtStart, 2, 2
```

Comment:
Is there any reason here for SwitchesOnlyAtStart ? It seems to me we could accept switches anywhere for this command.